package.json

```diff
@@ -42,6 +42,12 @@
     "npm": ">=6"
   },
   "config": {
-    "wp_org_slug": "woocommerce-google-analytics-integration"
+    "wp_org_slug": "woocommerce-google-analytics-integration",
+    "version_replace_paths": [
+      "src",
```

Comment:
It should be `includes` instead of `src`

`bin` seems not necessary as we are only adding sh files.

Reply:
Ofcourse, it should be `includes`. Not sure why I didn't replace that.